### ISSUE TITLE
onClientPlayerWasted but

### ISSUE THREAD

**Issue by 669344682** [multitheftauto/mtasa-blue#2516]
![image](https://user-images.githubusercontent.com/7806033/149668306-92c50e8a-4651-460e-ad30-90545087e2d7.png)
![image](https://user-images.githubusercontent.com/7806033/149668351-165dd0f9-0d56-464e-ad3a-a6f10c8424bf.png)
Background: client game speed will change to be slow when onClientPlayerWasted triggered

Besides, onClientPedWasted doesn't have any effects.

**Comment by ArranTuna** [comment]
Those screenshots don't show how the game speed has changed.

**Comment by Lpsd** [comment]
Which MTA version are you using? (F8 -> `ver`)

By "game speed" are you talking about FPS / lag or the GTA game speed? If the latter, please show the return value of [getGameSpeed](https://wiki.multitheftauto.com/wiki/GetGameSpeed) being changed.

What happens if you delay the setVisible call using a timer, does isPedDead return true? e.g

```lua
addEventHandler("onClientPlayerWasted", localPlayer, function()
    setTimer(setVisible, 1, 1, false) -- 1ms delay, try increasing if it doesn't work
end)
```

**Comment by PlatinMTA** [comment]
I think he is refering to the fact that ``isPedDead`` returns FALSE when the player just died (clientside at least). Probably using a timer will make ``isPedDead`` return true.

**Comment by Lpsd** [comment]
Yeah, that's what I'm trying to ascertain

**Comment by 669344682** [comment]
> 
Yes, you are right, I have changed the game speed on player wasted.
Does it will affect Judgment of death?

>
